### PR TITLE
V0.1 Movable Pulley Z Drive Mod

### PR DESCRIPTION
**Pull request by nhchiu**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
V0.1 Z Drive with Belt and Movable Pulley 
#### How was it tested? 
Printed and assembled. No issues after several successful prints.
#### Any background context you want to provide?
Nope. 
#### Screenshots (if appropriate)
Screenshots and photos are provided in the README of the mod.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Is `80T_gear.stl` identical to the one in the M4 repository?  If so, please remove the STL from your mod and point users to the M4 repository to get that STL, unmodified Voron parts should not be included in user mods.

`top_idler_{base,front}.stl` have unsupported overhangs at the bottom of the holes:

Screen Shot 2021-08-30 at 20 09 39

`top_idler_front.stl` also has an unsupported surface 3 layers up:
Screen Shot 2021-08-30 at 20 12 20

**Comment by nhchiu** [comment]
Thanks for your thorough review!

The 80T gear is modified to be a little thinner than the M4 one. The relevant note is added to the README.md.

Also the parts are updated with overhangs removed. The overhang at the hole for M5 hex nut is adjusted to match those on V2 XY joint.

File: printer_mods/nhchiu/V0.1_Movable_Pulley_Z/README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ There already are belt-driven mods like [theFPVGeek's V0 Z Belt mod](https://git
 - 4 x M3 Hex Nut
 - 3 x M5x20 BHCS/SHCS
 - 3 x M5 Hex Nut
+- 1 x M5 Washer (about 1mm thick, dosen't have to be accurate)
```
Comment (KiloQubit) [inline printer_mods/nhchiu/V0.1_Movable_Pulley_Z/README.md:26]: Replace `dosen't` with `doesn't`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates, just a minor typo in the README.

File: printer_mods/nhchiu/V0.1_Movable_Pulley_Z/README.md (hunk 1 above)
Comment (nhchiu) [reply printer_mods/nhchiu/V0.1_Movable_Pulley_Z/README.md]: Fixed now. Thanks!

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**